components/root_layout/root_layout_sticky.story.vue

```diff
@@ -1,35 +1,26 @@
 <template>
   <dt-root-layout
-    :body-class="bodyClass"
     :header-class="headerClass"
     :header-sticky="headerSticky"
-    :header-height="headerHeight"
     :content-class="contentClass"
-    :content-wrap-width-percent="contentWrapWidthPercent"
     :sidebar-class="sidebarClass"
     :sidebar-position="sidebarPosition"
-    :sidebar-width="sidebarWidth"
     :footer-class="footerClass"
-    :footer-height="footerHeight"
-    :fixed="fixed"
+    :fixed="false"
```

Comment:
why are we setting this variable, it'd be strange that you toggle it on the panel and it doesn't do anything.

Reply:
ah yeah thanks, was debugging something and forgot to change it back.